docs/extensibility/display-section.md

```diff
@@ -396,6 +397,32 @@ Text widgets render values as a simple text. This is the default behavior for al
 
 Block widgets are more complex layouts and you must use them only in the details body.
 
+### Alert
+
+Alert widgets display values using predefined types.
+
+#### Widget-specific parameters
+
+- **source** - information what has to be displayed.
```

Comment:
do we need this mention? `source` is the default for all components...
just an idea. maybe it makes sense to leave it...

Reply:
Is this **source** the same as mentioned in the [Display section](https://github.com/kyma-project/busola/blob/main/docs/extensibility/display-section.md#available-list-section-parameters)? If yes, then please use the same definition.